web/service/src/main/java/org/mobilitydata/gtfsvalidator/web/service/util/JobMetadata.java

```diff
@@ -17,7 +17,7 @@ public String getJobId() {
   }
 
   public void setJobId(String jobId) {
-    this.countryCode = countryCode;
+    this.jobId = jobId;
```

Comment:
suggestion: Use the following Lombok annotations to avoid issues in boilerplate code: @AllArgsConstructor, @Getter, and  @Setter.